arroyo-api/src/pipelines.rs

```diff
@@ -57,15 +60,24 @@ where
             Some(UdfLanguage::Rust) => {
                 schema_provider
                     .add_rust_udf(&udf.definition)
-                    .map_err(|e| bad_request(format!("Could not process UDF: {:?}", e)))?;
+                    .map_err(|e| anyhow!(format!("Could not process UDF: {:?}", e)))?;
```

Comment:
This function no longer uses those functions, but they're used elsewhere in the API, where I think they're useful because they give us some error message consistency.

Reply:
I don't quite follow what you're saying. I mean, previously we were able to distinguish between user errors (like invalid udf language) and system errors (like we failed to talk to the database), and now those all get collapsed into anyhow::error.